oidc-primer.html

```diff
@@ -0,0 +1,1219 @@
+    <li data-md>
+     <p><code>"typ": "dpop+jwt"</code>: The type of token. All DPoP Tokens should have a type of "dpop+jwt"</p>
+    <li data-md>
+     <p><code>"jwk": { "kty": "EC" ... }</code>: The client’s public key.</p>
+   </ul>
+   <p>Token Body:</p>
+<pre class="language-json highlight"><c- p>{</c->
+    <c- f>"htu"</c-><c- p>:</c-> <c- u>"https://auth.otherpod.example/token"</c-><c- p>,</c->
+    <c- f>"htm"</c-><c- p>:</c-> <c- u>"POST"</c-><c- p>,</c->
+    <c- f>"jti"</c-><c- p>:</c-> <c- u>"fb1264dd-fff1-4509-a7b1-0fe58d08d3e1"</c-><c- p>,</c->
+    <c- f>"iat"</c-><c- p>:</c-> <c- mi>1603389616</c->
+<c- p>}</c->
+</pre>
+   <ul>
+    <li data-md>
+     <p><code>"htu": "https://auth.otherpod.example/token"</code>: htu limits the token for
+use only on the given url.</p>
+    <li data-md>
+     <p><code>"htm": "POST"</code>: htm limits the token for use only on a specific http method, in this case <code>POST</code>.</p>
+    <li data-md>
+     <p><code>"jti": "fb1264dd-fff1-4509-a7b1-0fe58d08d3e1"</code>: jti is a unique identifier for the DPoP
+token that can optionally be used by the server to defend against replay attacks</p>
+    <li data-md>
+     <p><code>"iat": 1603389616</code>: The date the token was issued, in this case October 22, 2020 14:00:16
+GMT.</p>
+   </ul>
+   <h4 class="no-num heading settled" id="request-flow-step-4"><span class="content">4. Request Access Token</span><a class="self-link" href="#request-flow-step-4"></a></h4>
+   <p>Request:</p>
+<pre class="language-http highlight"><c- nf>POST</c-> <c- nn>/token</c-> <c- kr>HTTP</c-><c- o>/</c-><c- m>1.1</c->
+<c- e>Host</c-><c- o>:</c-> <c- l>auth.otherpod.example</c->
+<c- e>DPoP</c-><c- o>:</c-> <c- l>eyJhbGciOi...</c->
+...
+grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Auma-ticket
+&amp;ticket=016f84e8-f9b9-11e0-bd6f-0021cc6004de
+&amp;claim_token=eyj0...
+&amp;claim_token_format=http%3A%2F%2Fopenid.net%2Fspecs%2Fopenid-connect-core-1_0.html%23IDToken
+</pre>
+   <h4 class="no-num heading settled" id="request-flow-step-5"><span class="content">5. Checks ID Token expirations</span><a class="self-link" href="#request-flow-step-5"></a></h4>
+   <p>The AS checks if the id token is still valid by looking at the <code>exp</code> claim. If the token
+does not have an <code>exp</code> claim or the token is expired, the AS must reject the request with
+a 403 HTTP status. When using UMA 2.0, the AS will include <code>need_info</code> and <code>required_claims</code> values in the error response.</p>
+   <h4 class="no-num heading settled" id="request-flow-step-6"><span class="content">6. Checks the DPoP token url and method</span><a class="self-link" href="#request-flow-step-6"></a></h4>
+   <p>The AS checks the <code>htu</code> and <code>htm</code> claims of the DPoP token. If the <code>htu</code> does not match the
+protocol, origin and path of the request or the <code>htm</code> does not correspond the the http method
```

Comment:
By origin, does it have anything to do with CORS and friends, or should it rather be [authority](https://datatracker.ietf.org/doc/html/rfc3986#section-3.2)?

Reply:
It should be `scheme`, `authority` and `path`

Reply:
Indeed.